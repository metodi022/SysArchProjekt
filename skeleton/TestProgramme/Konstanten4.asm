lui $5, 0x4534
lui $5, 0x1423
sw $5, 4($0)
