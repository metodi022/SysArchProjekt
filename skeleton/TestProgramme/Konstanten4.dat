0x3c054534
0x3c051423
0xac050004
