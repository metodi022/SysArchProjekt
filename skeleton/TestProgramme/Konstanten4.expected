@5 14230000
